src/tincctl.c

```diff
@@ -3395,14 +3276,16 @@ int main(int argc, char *argv[]) {
 	orig_argc = argc;
 	tty = isatty(0) && isatty(1);
 
+	cmdline_conf = list_alloc((list_action_t)free_config);
```

Comment:
`cmdline_conf` is used inside `read_config_options`, which is called by `read_host_config`. I added this because otherwise, `read_host_config` would crash with a NULL pointer deref. I initially tried
```c
void read_config_options(splay_tree_t *config_tree, const char *prefix) {
	if(!cmdline_conf) {
		return;
	}
// ...
```
but thought it looked pretty ad-hoc.